dev-client/src/model/soilId/soilIdTypes.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {
+  SoilIdResults,
+  SoilIdStatus,
+} from 'terraso-client-shared/soilId/soilIdSlice';
+
+// TODO: This could be moved to client-shared
+export type SoilIdData = SoilIdResults & {status: SoilIdStatus};
```

Comment:
question: can we move this to client-shared now?

Reply:
Yep will do! It just seemed like some overhead that I didn't want it to block this PR